perceval/backends/core/pipermail.py

```diff
@@ -199,7 +201,7 @@ def fetch(self, from_date=DEFAULT_DATETIME):
 
         from_date = datetime_to_utc(from_date)
 
-        r = requests.get(self.url)
+        r = requests.get(self.url, verify=False)
```

Comment:
In my opinion, this should be an optional behaviour and configured passing the value to the class.